x/ecocredit/server/testsuite/suite.go

```diff
@@ -1038,6 +1047,86 @@ func (s *IntegrationTestSuite) TestScenario() {
 	})
 	s.Require().Nil(err)
 	s.Require().Equal(expectedSellOrderIds, createSellOrder.SellOrderIds)
+	orderId1 := createSellOrder.SellOrderIds[0]
+	orderId2 := createSellOrder.SellOrderIds[1]
+
+	// now we buy these orders
+	buyerAcc := acc5
+	expectedTotalCost := sdk.NewInt64Coin(coinPrice.Denom, 26_086_420)
```

Comment:
Might be nice to include the calculations here in a comment.

Reply:
added calculation comments in https://github.com/regen-network/regen-ledger/pull/1079/commits/2f1e43c799bef2167b427702fa541a9ced42ef04